src/Forcings/continuous_forcing.jl

```diff
@@ -127,8 +127,8 @@ Base.show(io::IO, forcing::ContinuousForcing{Nothing, Nothing, Nothing, P}) wher
         "├── parameters: $(forcing.parameters)", '\n',
         "└── field dependencies: $(forcing.field_dependencies)")
 
-Adapt.adapt_structure(to, forcing::ContinuousForcing{X, Y, Z}) where {X, Y, Z} =
-    ContinuousForcing{X, Y, Z}(Adapt.adapt(to, forcing.func),
+Adapt.adapt_structure(to, forcing::ContinuousForcing{LX, LY, LZ}) where {LX, LY, LZ} =
+    ContinuousForcing{LX, LY, LZ}(Adapt.adapt(to, forcing.func),
```

Comment:
Any specific reason why you changed `X` to `LX` throughout?

Reply:
@ali-ramadhan prefers LX, etc (stands for "x location"). "X" could be confused with the x-coordinate, perhaps. It's pervasive throughout the code but I feel it never hurts to pay off technical debt when you have the chance. Sisyphus at work...